src/graphql/resolvers/index.js

```diff
@@ -305,24 +301,34 @@ module.exports = function Resolvers (ssb) {
         const ids = await getFollowingIds(parent.id)
         return ids?.length
       },
-      // TODO: we need to be able to get alias by feedId in order for this to work
-      // for profiles that use the getProfile resolver
-      // this will only work for profiles found using the getProfileByAlias resolver
-      ssbURI: (parent) => {
-        const alias = parent.alias
-        if (!alias) return
+      ssbURI: async (parent) => {
+        const member = roomState.members.get(parent.id)
+        if (!member) return
+
+        // get their alias
+        const alias = member.aliases?.length
+         ? member.aliases[0]
+         : null
+         if (!alias) return
+
+        const aliasInfo = await getAlias(alias)
```

Comment:
getAlias(alias) is a bit confusing. Maybe rename getAliasInfo.

Not that crucial